pkg/vault/contracts/VaultFactory.sol

```diff
@@ -107,13 +111,14 @@ contract VaultFactory is ReentrancyGuardTransient, Ownable2Step {
             revert InvalidBytecode("VaultExtension");
         }
 
-        address vaultAddress = getDeploymentAddress(salt);
-        if (targetAddress != vaultAddress) {
+        // If this passes, we know we can use the targetAddress for the vaultAddress.
+        if (targetAddress != getDeploymentAddress(salt)) {
```

Comment:
```suggestion
        // If this passes, we know we can use the targetAddress for the vaultAddress.
        if (targetAddress != getVaultDeploymentAddress(salt)) {
```

Probably, if we change the function name, the comment is not needed.

Reply:
Yes; that's what I had to do to solve the stack-too-deep error